user/hardware/system-requirements.md

```diff
@@ -19,69 +19,51 @@ redirect_from:
+   - [Intel VT-d] or [AMD-Vi (also known as AMD IOMMU)]
+ - **Memory:** 16 GB RAM
+ - **Storage:** 128 GB free space
+   - High-speed solid-state drive strongly recommended
+ - **Graphics:** Intel integrated graphics processor (IGP) strongly recommended
+   - Nvidia GPUs may require significant [troubleshooting][nvidia]
+   - AMD GPUs have not been formally tested, but Radeons (especially RX580 and earlier) generally work well
+ - **Peripherals:** A non-USB keyboard or multiple USB controllers
+ - **TPM:** Trusted Platform Module (TPM) with proper BIOS support (required for [Anti Evil Maid])
+ - **Other:** Satisfaction of all [hardware certification requirements for Qubes 4.x]
 
- * Fast SSD (strongly recommended)
- * Intel IGP (strongly preferred)
-   * Nvidia GPUs may require significant [troubleshooting][nvidia].
-   * AMD GPUs have not been formally tested, but Radeons (RX580 and earlier) generally work well
-   * See the [Hardware Compatibility List]
- * [Intel VT-x] or [AMD-V] (required for running HVM domains, such as Windows-based AppVMs)
- * [Intel VT-d] or [AMD-Vi (aka AMD IOMMU)] (required for effective isolation of network VMs)
- * TPM with proper BIOS support (required for [Anti Evil Maid])
+## Choosing Hardware
 
-## Qubes Release 4.x ##
-
-### Minimum ###
-
- * 64-bit Intel or AMD processor (x86\_64 aka x64 aka AMD64)
- * [Intel VT-x] with [EPT] or [AMD-V] with [RVI]
- * [Intel VT-d] or [AMD-Vi (aka AMD IOMMU)]
- * 4 GB RAM
- * 32 GiB disk space
-
-### Recommended ###
-
- * Fast SSD (strongly recommended)
- * Intel IGP (strongly preferred)
-   * Nvidia GPUs may require significant [troubleshooting][nvidia].
-   * AMD GPUs have not been formally tested, but Radeons (RX580 and earlier) generally work well
-   * See the [Hardware Compatibility List]
- * TPM with proper BIOS support (required for [Anti Evil Maid])
- * A non-USB keyboard or multiple USB controllers
- * Also consider the [hardware certification requirements for Qubes 4.x].
-
-## Choosing Hardware ##
-
- * Please see the [Hardware Compatibility List] for a compilation of hardware reports generated and submitted by users across various Qubes versions.
+ - Please see the [Hardware Compatibility List] for a compilation of hardware reports generated and submitted by users across various Qubes versions.
    (For more information about the HCL itself, see [here][hcl-doc].)
- * See the [Certified Hardware] page.
- * See the [Hardware Testing] page.
+ - See the [Certified Hardware] page.
+ - See the [Hardware Testing] page.
 
-## Important Notes ##
+## Important Notes
 
- * **Installing Qubes in a virtual machine is not recommended, as it uses its own bare-metal hypervisor (Xen).**
- * Qubes **can** be installed on systems which do not meet the recommended requirements.
+ - **Installing Qubes in a virtual machine is not recommended, as it uses its own bare-metal hypervisor (Xen).**
+ - Qubes **can** be installed on systems which do not meet the recommended requirements.
```

Comment:
See https://github.com/QubesOS/qubes-doc/pull/1085#discussion_r524300888 - maybe we should include similar note here too?

Reply:
Actually, that comment is what prompted me to update this page.

I don't think it's necessary to include that note. That information is only of value to advanced technical users who wish to install 4.0 but not 4.1, which is a rapidly shrinking group of people, possibly soon to round to zero. If we document that somewhere, it should be on a page for advanced users, or simply here on GitHub, not in a place where everyone (including non-technical users) will have to read it and be confused by it.

Reply:
Yes, makes sense.